modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java

```diff
@@ -150,6 +159,34 @@ private static int _updateStartedAppServerStopCounters(
 		return originalCounter;
 	}
 
+	private Configuration _addConfigurationTestModules(final Project project) {
+		Configuration configuration = GradleUtil.addConfiguration(
+			project, TEST_MODULES_CONFIGURATION_NAME);
+
+		configuration.defaultDependencies(
+			new Action<DependencySet>() {
+
+				@Override
+				public void execute(DependencySet dependencySet) {
+					_addDependenciesTestModules(project);
```

Comment:
Do we always add?  I didn't see where we check to see if aires is already there.